usaspending_api/search/v2/views/spending_by_category_views/spending_by_category.py

```diff
@@ -203,8 +262,8 @@ def build_elasticsearch_search_with_aggregations(self, filter_query: ES_Q) -> Op
 
         return search
 
-    def query_elasticsearch_for_prime_awards(self, filter_query: ES_Q) -> list:
-        search = self.build_elasticsearch_search_with_aggregations(filter_query)
+    def query_elasticsearch_for_prime_awards(self, filter_query: ES_Q, spending_level: str) -> list:
```

Comment:
This should probably have a different name now that it is for transactions and prime awards.  Maybe just `query_elasticsearch`?

Reply:
Done